### PR TITLE
Feature/ch90637/doc guide 01 getting started

### PR DESCRIPTION
**Pull request by borja-munoz**
Initial version for the getting started guide with a very simple 3-step example

### CODE REVIEW

File: docs/guides/01-getting-started.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-In this guide, you will learn the basics of visualizing data with CARTO VL on top of a world [basemap](https://carto.com/help/glossary/#basemap). There are no previous requirements to complete this guide, but a basic knowledge of HTML, CSS and JavaScript would be helpful.
-
-After completing this guide, you will have your first CARTO VL map!
+After completing this guide, you will have your first Web SDK map!
 
 <div class="example-map">
     <iframe
         id="getting-started-final-result"
-        src="/developers/carto-vl/examples/maps/guides/getting-started/step-2.html"
+        src="/developers/web-sdk/examples/maps/guides/getting-started/step-2.html"
```
Comment (VictorVelarde) [inline docs/guides/01-getting-started.md]: Should this be the step-3 ?

**Review by VictorVelarde — APPROVED** [review]
Nice job! I just added some small editions. Check them and if you agree we're good to go.

File: docs/guides/01-getting-started.md (hunk 1 above)
Comment (borja-munoz) [reply docs/guides/01-getting-started.md]: Yes, corrected

**Comment by VictorVelarde** [comment]
Feel free to merge this in

**Pull request merged**